### ISSUE TITLE
See list of tx activity done by a address

### ISSUE THREAD

**Issue by bwann52** [MystenLabs/sui#1191]
On Accounts page, see info: hash, method, epoch, from, to, value, tx fee, status, time (if we have it)

**Comment by velvia** [comment]
We don't have a timestamp field in a transaction, but maybe this is motivation for adding one.... /cc @gdanezis @huitseeker

**Comment by Andrew47** [comment]
@666lcz --> so to unblock this, it looks like I'll need: 
* the name of the RPC API method `sui_*` that gives the list of transactions done by an address, 
* the arguments for this function (presumably this is just the address ID), 
* the expected type of the response and 
* whether to use `requestWithType` or `batchRequestWithType` (presumably `requestWithType`). 

I'll then be able to add the function in `src/providers/json-rpc-provider.ts` to connect to the RPC API.

**Comment by mystenmark** [comment]
Unless I'm misunderstanding, this is already done: https://github.com/MystenLabs/sui/blob/main/crates/sui-gateway/src/api.rs#L144-L150

**Comment by Andrew47** [comment]
The PR for updating the Sui Explorer to display transactions for an address is ready (https://github.com/MystenLabs/sui/pull/2276). 

@mystenmark --> thanks for the feedback. The aim of this issue is to update the frontend of the Sui Explorer website to display transactions for address using the new API functionality available in the Full Node.
